src/tools/ceph_kvstore_tool.cc

```diff
@@ -558,8 +558,13 @@ int main(int argc, const char *argv[])
     }
 
   } else if (cmd == "store-crc") {
-    uint32_t crc = st.traverse(string(), true, NULL);
-    std::cout << "store at '" << path << "' crc " << crc << std::endl;
+    if (argc < 4) {
+      usage(argv[0]);
+      return 1;
+    }
+    std::ofstream fs(argv[4]);
```

Comment:
it should be `argv[3]`. as cmd is `argv[2]`.

Reply:
it should be argv[4].  

```
➜  build git:(wip-kv-store-crc) ✗ ./bin/ceph-kvstore-tool bluestore-kv dev/osd2 store-crc t2.txt 2>/dev/null 
store at 't2.txt' crc 4037852868
```